delivery-metrics/src/delivery_metrics_config.py

```diff
@@ -0,0 +1,24 @@
+import time
+
+class DeliveryMetricsConfig:
+
+	def __init__(self, datestamp):
+
+		# path to sqlite db instance
+		self._DB_PATH = "../db/delivery_metrics.db"
```

Comment:
Future consideration: it might be good to parameterize this and a config attribute `conn_str` or something similar

Or at least to use `pathlib.Path(__file__).resolve().parent.parent` to get a stable reference to the root of the `deliverymetrics/` so that the script doesn't have to be run from within the `src/` sub-directory

Reply:
Thanks for the suggestion! I did not like that the user had to be in the src directory to run the script. This has been fixed in [7e86b67](https://github.com/agilesix/simpler-grants-sandbox/pull/48/commits/7e86b67a0ef7a27f5aa69e4fcab4416b0ab89085). 
Note: I used `os.path` instead of `pathlib` to support older versions of python (I'm on 3.2 and `pathlib` is available in 3.4)

Reply:
Gotcha, os.path works well too! 

When we migrate this to the main repo though, we'll probably want to pin the python version we're using and I'd be hesitant to use anything less than 3.10 especially because support for type hints weren't added until 3.6 and we'll definitely want to add those as we migrate the code over.